Makefile

```diff
@@ -95,8 +95,9 @@ test-unit: setup-envtest ## Run the unit tests
 	eval $$($(SETUP_ENVTEST) use -p env $(ENVTEST_VERSION)) && go test -tags $(GO_BUILD_TAGS) -count=1 -short $(UNIT_TEST_DIRS)
 
 FOCUS := $(if $(TEST),-v -focus "$(TEST)")
+IMAGE_REGISTRY := "testdata"
```

Comment:
super nit: should this be `?=` since we are setting the variable to `testdata` if no value is provided? 

https://www.gnu.org/software/make/manual/html_node/Setting.html

Reply:
Ah, yep you're totally right. Today I learned!